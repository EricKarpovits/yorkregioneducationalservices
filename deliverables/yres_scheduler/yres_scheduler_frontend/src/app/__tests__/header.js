import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/header';

it('renders properly', () => {
    const component = renderer.create(<Header/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});