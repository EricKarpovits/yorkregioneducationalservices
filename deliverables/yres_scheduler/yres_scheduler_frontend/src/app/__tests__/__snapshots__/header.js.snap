// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-light"
  id="nav-header"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="#home"
      id="welcome"
    >
      Welcome, Admin
    </a>
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        className="ms-auto navbar-nav nav-tabs"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rr-ui-event-key="#home"
          disabled={false}
          href="#home"
          onClick={[Function]}
        >
          Profiles
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key="#flooplan"
          disabled={false}
          href="#flooplan"
          onClick={[Function]}
        >
          FloorPlan
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key="#schedules"
          disabled={false}
          href="#schedules"
          onClick={[Function]}
        >
          View Schedules
        </a>
      </div>
    </div>
  </div>
</nav>
`;
